.github/workflows/zip.yml

```diff
@@ -7,6 +7,8 @@ on:
     - '.github/workflows/zip.yml'
     - 'scripts/build_non_firebase_sdks.sh'
     - 'Gemfile*'
+    # DELETE BEFORE pushing
+    - 'FirebaseCore.podspec'
```

Comment:
By "pushing", are you referring to publishing pods?

Reply:
#12434